app/main.py

```diff
@@ -1,5 +1,81 @@
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: float) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: float,
+            coords: list = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+    def go_forward(self, step: int = 1) -> list:
+        one_step_move = self.coords
+        one_step_move[1] += 1 if step is None else step
+        return one_step_move
+
+    def go_back(self, step: int = 1) -> list:
```

Comment:
```suggestion
    def go_back(self, step: int = 1) -> list[int]:
```
use more descriptive type annotation